src/javascript/app_2/Modules/Trading/Containers/trade.jsx

```diff
@@ -23,6 +23,7 @@ class Trade extends React.Component {
     render() {
         const contract_id = getPropertyValue(this.props.purchase_info, ['buy', 'contract_id']);
         const form_wrapper_class = this.props.is_mobile ? 'mobile-wrapper' : 'sidebar-container desktop-only';
+        const showLastDigitStats = this.props.market === 'volidx';
```

Comment:
same concern with the variable name mentioned above

Reply:
also as mentioned in trello, we don't want to show this for all volidx. we want to show it for all trade types of digit, e.g. digit match/diff, digit even/odd, and digit over/under